src/response-data.zig

```diff
@@ -0,0 +1,41 @@
+alloc: Allocator,
+map: std.StringHashMap(*anyopaque),
+
+pub const ResponseData = @This();
```

Comment:
I don't think we should export this. It gets weird with when using ResponseData it will endless show a ResponseData field as accessible.

Reply:
that's a very good point